services/tenant-ui/frontend/src/helpers/index.ts

```diff
@@ -24,14 +24,24 @@ export function formatDate(value: string) {
 
 /**
  * @function formatDateLong
- * Converts a date to an 'MMMM D yyyy, h:mm:ss a' formatted string
+ * Converts an acapy-returned string date to an 'MMMM D yyyy, h:mm:ss a' formatted string
  * @param {String} value A string representation of a date
  * @returns {String} A string representation of `value`
  */
 export function formatDateLong(value: string) {
   return _dateFnsFormat(value, 'MMMM d yyyy, h:mm:ss a');
 }
 
+/**
+ * @function formatUnixDate
+ * Converts a unix time number to an 'MMMM D yyyy' formatted string
+ * @param {String} value A unix timestamp number
+ * @returns {String} A string representation of `value`
+ */
+export function formatUnixDate(value: number) {
+  return format(fromUnixTime(value), 'MMMM d yyyy');
```

Comment:
Could  we put the date format in the i18n file, so it can be localized?
Same for other, pre-existing values.

Reply:
can use date-fns/locale in these I think, adding https://github.com/bcgov/traction/issues/593